### PR TITLE
Dockerize snapshot script

### PR DESCRIPTION
**Pull request by goruha**
## What
* Created script that make ebs snapshots based on pod volumes

## Why
* EBS snapshots is good way to create backups

### CODE REVIEW

File: rootfs/bin/create-snapshot [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
```
Comment (osterman) [inline rootfs/bin/create-snapshot:2]: should `set -o pipefail` https://bclary.com/blog/2006/07/20/pipefail-testing-pipeline-exit-codes/

File: rootfs/bin/create-snapshot [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+export DESCRIPTION=${DESCRIPTION:-"This is snapshot create automaticaly by kube-ebs-snapshot"}
```
Comment (osterman) [inline rootfs/bin/create-snapshot]: `This snapshot was created automaticaly by kube-ebs-snapshot`

File: rootfs/bin/create-snapshot [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+fi
+
+if [[ $NAME == "all" || $NAME == "" ]];
+then
+  NAME_FILTER="*"
+else
+  NAME_FILTER="?(@.name==\"$NAME\")"
+fi
+
+echo "Creating snapshots based on PVC"
```
Comment (osterman) [inline rootfs/bin/create-snapshot]: `echo "Creating snapshots based on $NAME PVC in the $NAMESPACE namespace"`

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ARG K8S_VERSION=v1.5.6
+
+ENV DESCRIPTION=
+ENV EXTERNAL_ARGS=
+ENV TAGS=
+
+RUN apk update \
+    && apk add unzip \
+          python make bash jq \
+          bash-completion && \
```
Comment (osterman) [inline Dockerfile]: Can drop the support for `bash-completion`

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+ADD https://storage.googleapis.com/kubernetes-release/release/$K8S_VERSION/bin/linux/amd64/kubectl /usr/local/bin/kubectl
+RUN chmod +x /usr/local/bin/kubectl
+
+# Install aws cli bundle
+ADD https://s3.amazonaws.com/aws-cli/awscli-bundle.zip ./awscli-bundle.zip
+RUN unzip awscli-bundle.zip \
+    && ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws \
+    && rm awscli-bundle.zip \
+    && rm -rf awscli-bundle \
+    && ln -s /usr/local/aws/bin/aws_bash_completer /etc/bash_completion.d/aws.sh \
```
Comment (osterman) [inline Dockerfile]: and drop this

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,13 @@
+SHELL = /bin/bash
```
Comment (osterman) [inline Makefile]: Use the short notation? https://github.com/cloudposse/build-harness#usage

**Review by osterman — CHANGES_REQUESTED** [review]
See inline comments

File: Dockerfile (hunk 4 above)
Comment (goruha) [reply Dockerfile]: Done

File: Dockerfile (hunk 5 above)
Comment (goruha) [reply Dockerfile]: Done

File: Makefile (hunk 6 above)
Comment (goruha) [reply Makefile]: Done

File: rootfs/bin/create-snapshot (hunk 1 above)
Comment (goruha) [reply rootfs/bin/create-snapshot:2]: Done

File: rootfs/bin/create-snapshot (hunk 2 above)
Comment (goruha) [reply rootfs/bin/create-snapshot]: Done

File: rootfs/bin/create-snapshot (hunk 3 above)
Comment (goruha) [reply rootfs/bin/create-snapshot]: Done

**Review by osterman — APPROVED** [review]

**Pull request merged**